frontend/src/components/common/spinner/spinner.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'common/types/types';
+import styles from './styles.module.scss';
+
+type Props = {
+  isOverflow?: boolean;
+};
+
+const Spinner: FC<Props> = ({ isOverflow = false }) =>
+  isOverflow ? (
+    <div className={styles.container}>
+      <div className={styles.loader}></div>
+    </div>
+  ) : (
+    <div className={styles.loader}></div>
+  );
```

Comment:
Just to make it more readable
```suggestion
const Spinner: FC<Props> = ({ isOverflow = false }) => {
  if (isOverflow) {
    return (
      <div className={styles.container}>
        <div className={styles.loader}></div>
      </div>
    );

    return (
      <div className={styles.loader}></div>
    );
};
```